package/yast2-ycp-ui-bindings.spec

```diff
@@ -16,11 +16,11 @@
 #
 
 # YUIWidget_CustomStatusItemSelector
-%define min_yui_version	3.9.1
-%define yui_so		11
+%define min_yui_version	3.10.0
+%define yui_so		12
 
 Name:           yast2-ycp-ui-bindings
-Version:        4.2.9
+Version:        4.2.10
```

Comment:
As this is for TW/SLE15-SP3 only the version should be 4.3.0...

Reply:
Done.